syntax = "proto3";

package api.unsafe.sensor_void_energy;

service SensorVoidEnergyServer {
    rpc read_sensor_data(Void) returns (SensorData);
}

message Void {
}

message SensorData {
    string hexdata = 1;
}